Sources/SwifterSwift/UIKit/UITextFieldExtensions.swift

```diff
@@ -144,18 +144,44 @@ public extension UITextField {
         leftViewMode = .always
     }
 
+    /// SwifterSwift: Add padding to the right of the textfield rect.
+    ///
+    /// - Parameter padding: amount of padding to apply to the right of the textfield rect.
+    func addPaddingRight(_ padding: CGFloat) {
```

Comment:
Any chance **this `func`** and the above **pre-existing** one could be renamed to ~`add`~**`set`**`PaddingRight` instead?
It seems to completely ignore any pre-existing `rightView` as well as its `.frame`...

Reply:
Thank you for your suggestion, but I just added an `addPaddingRight` on the basis of the previous, and its pre-existing `rightView` is nil.
So I think it is still using `add`.

Reply:
@rypyak you're right, but renaming the function above would be a breaking change, which we don't want to do. I think better to be consistent here.